cardano-cli/src/Cardano/CLI/Shelley/Run/KeyGen.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StrictData #-}
+
+{-# OPTIONS_GHC -Wno-all-missed-specialisations #-}
+
+module Cardano.CLI.Shelley.Run.KeyGen
+  ( runGenesisKeyGenGenesis
+  , runGenesisKeyGenDelegate
+  , runGenesisKeyGenUTxO
+  , runGenesisKeyHash
+  , runGenesisVerKey
+  , runColdKeyGen
+  ) where
+
+import           Cardano.Prelude hiding (option, trace)
+
+import qualified Data.ByteString.Char8 as BS
+
+import           Control.Monad.Trans.Except (ExceptT)
+import           Control.Monad.Trans.Except.Extra (firstExceptT)
+
+import qualified Shelley.Spec.Ledger.Keys as Ledger
+import qualified Cardano.Crypto.Hash.Class as Crypto
+
+import           Cardano.CLI.Key (VerificationKeyFile(..))
+import           Cardano.CLI.Ops (CliError (..))
+import           Cardano.CLI.Shelley.Parsers
+
+import           Cardano.Config.Shelley.ColdKeys
+import           Cardano.Config.Shelley.OCert
+import           Cardano.Config.Types (SigningKeyFile(..))
+
+
+
+runGenesisKeyGenGenesis :: VerificationKeyFile -> SigningKeyFile
+                        -> ExceptT CliError IO ()
+runGenesisKeyGenGenesis = runColdKeyGen GenesisKey
+
+
+runGenesisKeyGenDelegate :: VerificationKeyFile
+                         -> SigningKeyFile
+                         -> OpCertCounterFile
+                         -> ExceptT CliError IO ()
+runGenesisKeyGenDelegate vkeyPath skeyPath (OpCertCounterFile ocertCtrPath) = do
+    runColdKeyGen (OperatorKey GenesisDelegateKey) vkeyPath skeyPath
```

Comment:
Just pointing out that there are 4 spaces here and 2 spaces elsewhere in the code base. The prevailing spacing is 2 spaces.

Reply:
That was actually code in your commit :).